### ISSUE TITLE
The meaning of arg numTrainingImages

### ISSUE THREAD

**Issue by dreamPoet** [NVlabs/planercnn#9]
Hi,

I find you set an arg called numTrainingImages, which is "the number of images to train". However, in the real training, I find it only influence the step of saving learned variables instead of the images used for train, which is fixed as 1400647. Is that right?

**Comment by art-programmer** [comment]
Yes, that's correct. It should be called numTrainingImagesPerEpoch. The total number of training images 1400647. It's kind of inaccurate as I trained only for 20 epochs, which means not all the images are used.

**Comment by dreamPoet** [comment]
> Yes, that's correct. It should be called numTrainingImagesPerEpoch. The total number of training images 1400647. It's kind of inaccurate as I trained only for 20 epochs, which means not all the images are used.

Sorry I am not sure if I understand your meaning or not. In your paper you said "train the network ... for 10 epochs with 100,000 randomly sampled images"... but you said you set each epoch as 1000 images and only run 20 epoch? And there is also a parameter called numEpochs in options.py set as 1000...(in fact, in config.py, there is also a STEPS_PER_EPOCH..) could you give me more details about differences betweens them?

**Comment by dreamPoet** [comment]
I think one epoch means a complete train on the whole used training dataset, and a step means a training step on a batch of training dataset. Here, the used training dataset is 1400647, the step is 1400647/1000, and the epoch is 10. But you means the epoch is 20 and the training set is 20*1000?

**Comment by yangengt123** [comment]
Hi @art-programmer , 
I have a similar question. In the paper, you mentioned you train "for 10 epochs with 100,000 randomly sampled images", which I assume you randomly pick 100,000 images per epoch and run the code for 10 epochs, but based on the default parameters you set, it seems you only use 1000 images per epoch and train the network for 1000 epochs.
May I ask what is the right setting to reproduce the result in the paper? 
Thanks in advance.

**Comment by cgarg-tud** [comment]
Hi @dreamPoet  did you figure out the final parameters for training ?

**Comment by dreamPoet** [comment]
> Hi @dreamPoet did you figure out the final parameters for training ?

Nope.

**Comment by ajithvcoder** [comment]
@dreamPoet  could u give the link for few annotations for training if you have ,      the link in repo is broken
